### PR TITLE
Dynamic log level config

### PR DESCRIPTION
**Pull request by davewalter**
Update the docs for the new dynamic log-level feature (see https://jira.eng.vmware.com/browse/TME-2141).

Co-authored-by: Ashwin Krishna <krishnaas@vmware.com>

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -39,16 +40,75 @@ The following is a brief description of specific Application Service Adapter dep
 6. The `korifi-job-task-runner-controller-manager` deployment is tasked with handling all task-related functionality.
 7. The `cartographer-builder-runner-controller-manager` deployment is tasked with creating Cartographer workloads for apps when using the experimental Cartographer builder/runner flow.
 
+#### Dynamically changing component logging level
+
+While it is possible to <a href="component-log-level">change the logging level</a> for each component of Application Service Adapter during an upgrade, this will result in the component being restarted.
+
+To avoid these component restarts, you can temporarily change the logging level for a specific component by modifying the `logLevel` setting in the corresponding ConfigMap.
+
+Before you perform any customization to the ConfigMaps, you must prevent the platform from overwriting your changes with the desired state from your original install values.
```
Comment (tcdowney) [inline troubleshooting.md:49]:
```suggestion
Before you perform any customization to the ConfigMaps, you must prevent the platform from reverting your changes back to their original install values.
```

**Review by tcdowney — APPROVED** [review]
Overall LGTM, with one minor wording suggestion.

File: troubleshooting.md [hunk 2]
```diff
@@ -39,16 +40,75 @@ The following is a brief description of specific Application Service Adapter dep
 6. The `korifi-job-task-runner-controller-manager` deployment is tasked with handling all task-related functionality.
 7. The `cartographer-builder-runner-controller-manager` deployment is tasked with creating Cartographer workloads for apps when using the experimental Cartographer builder/runner flow.
 
+#### Dynamically changing component logging level
+
+While it is possible to <a href="component-log-level">change the logging level</a> for each component of Application Service Adapter during an upgrade, this will result in the component being restarted.
```
[the remark this answers is not in the document]
Comment (davewalter) [reply troubleshooting.md]: Fixed. 😅

File: troubleshooting.md (hunk 1 above)
Comment (davewalter) [reply troubleshooting.md]: Updated. 👍

**Pull request merged**